### PR TITLE
docs: add static column width demo to guides

### PR DESCRIPTION
**Pull request by gsobolev**

### CODE REVIEW

**Comment by SergeyAlexeev** [comment]
Do we expect horizontal scrollbar in a demo?

File: packages/dx-react-demos/src/demo-sources/grid-basic/static-column-width.jsxt [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        { name: 'product', title: 'Product' },
+        { name: 'amount', title: 'Sale Amount' },
+      ],
+      rows: generateRows({ columnValues: globalSalesValues, length: 8 }),
+      tableColumnExtensions: [
+        { columnName: 'region', width: 180 },
+        { columnName: 'sector', width: 130 },
+        { columnName: 'customer', width: 240 },
+        { columnName: 'product', width: 210 },
+        { columnName: 'amount', width: 140 },
```
Comment (MaximKudriavtsev) [inline packages/dx-react-demos/src/demo-sources/grid-basic/static-column-width.jsxt]: Maybe we should display the `align` property too?

**Comment by gsobolev** [comment]
I don't think horizontal scrollbar is a problem here. It may also emphasize that columns preserve widths specified in the config.

File: packages/dx-react-demos/src/demo-sources/grid-basic/static-column-width.jsxt (hunk 1 above)
Comment (gsobolev) [reply packages/dx-react-demos/src/demo-sources/grid-basic/static-column-width.jsxt]: I believe we should not mix two demos

**Pull request merged**